sections/main-product.liquid

```diff
@@ -296,7 +296,7 @@
     <div class="product__info-wrapper grid__item{% if settings.page_width > 1400 and section.settings.media_size == "small" %} product__info-wrapper--extra-padding{% endif %}">
       <div
         id="ProductInfo-{{ section.id }}"
-        class="product__info-container{% if section.settings.enable_sticky_info %} product__info-container--sticky{% endif %}"
+        class="color-{{ section.settings.container_color_scheme }} product__info-container{% if section.settings.enable_sticky_info %} product__info-container--sticky{% endif %}"
```

Comment:
small nitpick, I would put the `color-{{ section.settings.container_color_scheme }}` class at the end of the class list since it's more generic and it follows what you did above 👍